packages/cfl/tests/test_finding.py

```
@@ -0,0 +1,917 @@
+        "challenge",
+        1,
+        title="Style nit",
+        severity="LOW",
+        visibility="presented",
+    )
+
+    captured = capsys.readouterr()
+    assert "unknown_severity" in captured.err
+
+
+def test_record_finding_unknown_severity_still_writes_row(db_conn, capsys):
```

Ruff (0.16.1)

[warning] Unused function argument: `capsys`

(ARG001)